README.md

```diff
@@ -321,6 +321,20 @@ cur.execute('SELECT * FROM system.runtime.nodes')
 rows = cur.fetchall()
 ```
 
+## Roles
+
+Authorization roles to use for catalogs, specified as a dict with key-value pairs for the catalog and role. For example, `{"catalog1": "roleA", "catalog2": "roleB"}` sets roleA for catalog1 and roleB for catalog2.
```

Comment:
```suggestion
Authorization roles to use for catalogs, specified as a dict with key-value pairs for the catalog and role. For example, `{"catalog1": "roleA", "catalog2": "roleB"}` sets `roleA` for `catalog1` and `roleB` for `catalog2`.
```

Also might be useful to link to Trino docs about roles since AFAIK there is also something called a system role.

Reply:
There seems no specific page about roles, only about the SQL syntax. I think a proper page about how roles work in Trino will be definitely beneficial.

Reply:
cc: @mosabua @dain regarding some discussion of what roles mean - we can just explain the concept because AFAIK there is no plugin providing roles in Trino - just the SPI.